### ISSUE TITLE
Add more descriptive comments to citations for CohesiveEnergy

### ISSUE THREAD

**Issue by computron** [hackingmaterials/matminer#145]
@saurabh02 

See this todo:
```
# TODO: @sbajaj unclear whether cohesive energies are taken from first ref, second ref, or combination of both
```
matminer.featurizers.composition.CohesiveEnergy#citations

You can add some code comments that explains what is going on. This could be part of the main doc for the CohesiveEnergy featurizer, e.g.
```
        Class to get cohesive energy per atom of a compound by adding known
        elemental cohesive energies from the formation energy of the
        compound. <<SOME MORE DESCRIPTION HERE ABOUT KNOWN COHESIVE ENERGIES SOURCE>>
```

**Comment by computron** [comment]
See also the comment:

```TODO: Where is this data from? -wardlt```

matminer/utils/data.py:103

You can put the description in the docstring for the ```CohesiveEnergyData``` class

**Comment by computron** [comment]
@saurabh02 Basically we just want to know which of these two sources, or if it was a combination of both, that you got the cohesive energy data file from:

        return [
            "@misc{, title = {{Knowledgedoor Cohesive energy handbook}}, "
            "url = {http://www.knowledgedoor.com/2/elements{\_}handbook/cohesive{\_}energy.html}}",
            "@book{Kittel, author = {Kittel, C}, isbn = {978-0-471-41526-8}, "
            "publisher = {Wiley}, title = {{Introduction to Solid State "
            "Physics, 8th Edition}}, year = {2005}}"]

If it was both, how did you handle entries that were in both sources? i.e., did you prefer one source over the other, average them etc?

**Comment by saurabh02** [comment]
@computron sorry i missed seeing the notification for this.

**Comment by computron** [comment]
Got, it thanks! Resolved via: 	481f5b66b18a7704f0327b1ddc26b8cb40de72a4
